c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -498,10 +497,6 @@ $(APP_OUTPUT_DIR)/build.min.css: $(LESS_FILES) .build/node_modules.timestamp
 	mkdir -p $(dir $@)
 	./node_modules/.bin/lessc --clean-css $(PACKAGE)/static-ngeo/less/$(PACKAGE).less $@
 
-$(APP_OUTPUT_DIR)/build.css: $(LESS_FILES) .build/node_modules.timestamp
-	mkdir -p $(dir $@)
-	./node_modules/.bin/lessc $(PACKAGE)/static-ngeo/less/$(PACKAGE).less $@
-
```

Comment:
Just a remark, to be as we are with `cgxp` we shouldn't use the `--clean-css` when `DEVELOPMENT` is set to `TRUE`...